platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/UnsignedEvent.java

```diff
@@ -151,6 +159,17 @@ public BaseEventHashedData(
         this.allParents = createAllParentsList();
         this.birthRound = birthRound;
         this.timeCreated = Objects.requireNonNull(timeCreated, "The timeCreated must not be null");
+
+        this.payloads = transactions.stream().map(EventPayload::new).toList();
+        this.eventCore = new EventCore(
+                creatorId.id(),
+                birthRound,
```

Comment:
since birth round is now part of EventCore, it does not have to be a separate variable

Reply:
done